lib/routers/establishment/project-versions.js

```diff
@@ -1,7 +1,9 @@
 const { Router } = require('express');
 const isUUID = require('uuid-validate');
 const { permissions } = require('../../middleware');
-const { NotFoundError } = require('../../errors');
+const { NotFoundError, BadRequestError } = require('../../errors');
+
+const perms = task => permissions(task, (req, res) => ({ licenceHolderId: req.project.licenceHolderId }));
```

Comment:
I'm mildly surprised that lint hasn't 🐄-ed about the unused `res` here.

Reply:
hmm me too